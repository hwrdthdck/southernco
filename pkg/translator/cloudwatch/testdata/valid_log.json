{
  "timestamp": 1725544035523,
  "message": "Hello world!"
}