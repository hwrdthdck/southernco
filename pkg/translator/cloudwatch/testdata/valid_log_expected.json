{
  "resourceLogs":[
    {
      "resource":{

      },
      "scopeLogs":[
        {
          "scope":{

          },
          "logRecords":[
            {
              "timeUnixNano":"1725544035523000000",
              "body":{
                "stringValue":"Hello world!"
              },
              "traceId":"",
              "spanId":""
            }
          ]
        }
      ]
    }
  ]
}